metadata/contributors.json

```diff
@@ -5446,6 +5446,15 @@
          "macpherson"
       ]
    },
+   {
+      "emails" : [
+         "lhernandez28@apple.com",
+         "lizette4662@gmail.com"
+      ],
+      "github" : "liz4662",
+      "name" : "Lizette Hernandez",
+      "status" : "contributer"
```

Comment:
I think default state itself is `contributor`, so you don't need to add this line.